gadget/ondisk_test.go

```diff
@@ -423,9 +464,11 @@ func (s *ondiskTestSuite) TestBuildPartitionList(c *C) {
 	// the expected expanded writable partition size is:
 	// start offset = (2M + 1200M), expanded size in sectors = (8388575*512 - start offset)/512
 	sfdiskInput, create := gadget.BuildPartitionList(dl, pv)
-	c.Assert(sfdiskInput.String(), Equals, `/dev/node3 : start=     2461696, size=     5926879, type=0FC63DAF-8483-4772-8E79-3D69D8477DE4, name="Writable", attrs="GUID:59"
+	c.Assert(sfdiskInput.String(), Equals,
+		`/dev/node3 : start=     2461696, size=      262144, type=0FC63DAF-8483-4772-8E79-3D69D8477DE4, name="Save", attrs="GUID:59"
```

Comment:
Just a reminder that partitions with attribute 59 currently won't survive reinstalls, but this will probably be addressed later.

Reply:
@bboozzoo  we'll need to think about this, we might just set it to mean managed by install, but then decide what to drop/keep separately also considering the fs label, possibly?  anyway if there's no fs label there is probably not data to preserve either... but open to input here

Reply:
Agreed. I think we should have all the necessary bits to make a decision inside `install.Run()`.